e2e/support/commands/user/authentication.ts

```diff
@@ -1,5 +1,18 @@
 import { USERS } from "e2e/support/cypress_data";
 
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      signIn: (user?: keyof typeof USERS) => void;
```

Comment:
For correctness, the return type would be `Cypress.Chainable<Cypress.Response<any>>` because we could chain it. There are some usage that we chain from `cy.signIn*` so technically the result isn't void.

https://github.com/metabase/metabase/blob/780e63246405e6ec3cbdd015a2742f6924311d06/e2e/test/scenarios/onboarding/notifications.cy.spec.js#L63